lxd/storage/drivers/driver_dir_utils.go

```diff
@@ -12,6 +12,15 @@ import (
 	"github.com/lxc/lxd/shared/units"
 )
 
+// withoutGetVolID returns a copy of this struct but without the volId property set.
```

Comment:
@stgraber just a tiny nit, but the argument not being set is `volIDFunc` not `volId` here.

Reply:
fixed